hub/core/compression/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .base import BaseNumCodec, BaseImgCodec
+from .jpeg import JPEG
+from .png import PNG
+from .webp import WEBP
+from .numpy import NUMPY
+from .lz4 import LZ4
+from .zstd import ZSTD
+
+AVAILABLE_COMPRESSORS = [mod for mod in dir() if mod.isupper()]
```

Comment:
so you do have a variable here, i suggest using this in the tests. also, mod.isupper is being used, why?

Reply:
To filter compressor names from dir() that returns all attrs of file.